sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -21,6 +21,7 @@ import java.sql.{Date, Timestamp}
 import java.util.TimeZone
 
 import scala.util.Random
+import scala.language.implicitConversions
```

Comment:
```
Warning:Warning:line (39)implicit conversion method stringToUTF8Str should be enabled
by making the implicit value scala.language.implicitConversions visible.
This can be achieved by adding the import clause 'import scala.language.implicitConversions'
or by setting the compiler option -language:implicitConversions.
See the Scaladoc for value scala.language.implicitConversions for a discussion
why the feature should be explicitly enabled.
  implicit def stringToUTF8Str(str: String): UTF8String = UTF8String.fromString(str)
```